### ISSUE TITLE
Use correct biological terminology

### ISSUE THREAD

**Issue by ekkolon** [serengil/deepface#645]
Hey @serengil!

Nice to see you still actively working on this library!

I would like to propose a change in terminology for the following occurrences: 
- `Race` -> `Ethnicity` and `race` -> `ethnicity`, respectively

The PR should not introduce changes in functionality but rather just
rename entities and docs that use the above terms.

Let me know if I can put my hands on a PR to implement this!

**Comment by serengil** [comment]
PRs are welcome

**Comment by ekkolon** [comment]
> PRs are welcome

_This is not a political debate, nor am I intending to start one. I just wanted to lay out why I changed my mind and feel hesitant to make a PR that changes the current terminology._ 

I came to realize that making these changes would probably mislead users by implying that within the context of recognizing physical characteristics, *race* would be synonymous to *ethnicity*, as per definition.

In fact, if we go along with one of the definition of ethnicity (the differentiation of groups of people according to cultural expression and identification) it would go beyond the scope of what the **Race** model does, namely recognizing similar physical and biological characteristics and the differentiation of those.

This closed issue #621 by @MrGranddy pointed out the term race to be incorrect for classifying humans. So what would be the correct term to use, considering the scope and function of the model? Honestly, I don't know. Constructive feedback is greatly appreciated!
